### PR TITLE
Fix avatar

### PR DESCRIPTION
**Pull request by HSalmon**

### CODE REVIEW

File: lineman/app/components/user_avatar/user_avatar.haml [hunk 1]
```diff
@@ -1,7 +1,5 @@
-.user-avatar{aria-hidden: 'true', class: '{{size}}'}
-  .avatar.avatar-initials{ng-if: 'user.avatarKind == "initials"', ng-class: '{ coordinator: coordinator }'}
+.user-avatar{aria-hidden: 'true', ng-class: '{"user-avatar--large": large, "user-avatar--small": small, "user-avatar--coordinator": coordinator}'}
```
Comment (gdpelican) [inline lineman/app/components/user_avatar/user_avatar.haml]: ng-class is way overkill here imho, especially given that we're going to want to add more sizes and that the directive doesn't need to be dynamic based on size.

```
$scope.size = 'small' unless $scope.size?
```
```
.user-avatar{class: 'user-avatar--{{size}}'}
```

File: lineman/app/components/user_avatar/user_avatar.scss [hunk 2]
```diff
@@ -1,31 +1,50 @@
-.avatar-initials{
-  @include smallIcon;
+// .avatar-initials{
```
Comment (gdpelican) [inline lineman/app/components/user_avatar/user_avatar.scss]: Can we remove? Also I don't see a replacement for the user-avatar coordinator class?

File: lineman/app/components/user_avatar/user_avatar.haml (hunk 1 above)
Comment (robguthrie) [inline lineman/app/components/user_avatar/user_avatar.haml]: thank you.. we went through a bunch of css contortions and there are still knots to iron out. This is a good suggestion

**Comment by HSalmon** [comment]
@gdpelican, @robguthrie, is this good to go?

**Comment by gdpelican** [comment]
Hey Hannah, we'll need a smaller avatar ('tiny'?) for the profile picture in the top right of the navbar.
screen shot 2015-07-19 at 8 08 48 pm

**Comment by gdpelican** [comment]
This looks good to me :ship:

**Pull request merged**